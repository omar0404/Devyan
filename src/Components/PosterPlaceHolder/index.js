import PosterPlaceHodler from './PosterPlaceHolder';
export default PosterPlaceHodler;
