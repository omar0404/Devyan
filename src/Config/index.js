export const POSTER_BASE_URL = 'http://image.tmdb.org/t/p/w500';
