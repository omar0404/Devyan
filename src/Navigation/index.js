import React from 'react';
import {NavigationContainer} from '@react-navigation/native';
import HomeScreens from './HomeScreens';

const RootNavigation = () => (
  <NavigationContainer>
    <HomeScreens />
  </NavigationContainer>
);
export default RootNavigation;
