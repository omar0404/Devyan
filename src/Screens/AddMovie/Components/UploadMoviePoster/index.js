import UploadMoviePoster from './UploadMoviePoster';
export default UploadMoviePoster;
