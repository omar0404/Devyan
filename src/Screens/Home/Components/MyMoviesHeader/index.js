import MyMoviesHeader from './MyMoviesHeader';
export default MyMoviesHeader;
