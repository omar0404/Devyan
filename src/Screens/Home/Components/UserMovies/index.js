import UserMovies from './UserMovies';
export default UserMovies;
