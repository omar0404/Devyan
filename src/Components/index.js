import Section from './Section';
import PosterPlaceHolder from './PosterPlaceHolder';
import MovieCard from './MovieCard';
export {MovieCard, Section, PosterPlaceHolder};
