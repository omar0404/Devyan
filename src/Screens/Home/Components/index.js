import MyMovies from './MyMovies';
import AllMovies from './AllMovies';
import UserMovies from './UserMovies';
import MyMoviesHeader from './MyMoviesHeader';
export {MyMovies, AllMovies, UserMovies, MyMoviesHeader};
