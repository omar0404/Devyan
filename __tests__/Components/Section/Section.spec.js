import 'react-native';
import React from 'react';
import {Section} from 'MoviesApp/src/Components';

import renderer from 'react-test-renderer';

describe('Section Componennt', () => {
  it('renders correctly', () => {
    const SectionComponent = renderer.create(<Section />).toJSON();
    expect(SectionComponent).toMatchSnapshot();
  });
});
