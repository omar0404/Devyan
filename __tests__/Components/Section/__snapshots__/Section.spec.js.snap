// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Componennt renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "height": "auto",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#991F25",
        "height": 15,
        "width": 5,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "rgb(255, 255, 255)",
        "fontSize": 18,
        "marginLeft": 8,
      }
    }
  />
</View>
`;
